tests/get_account_test.go

```diff
@@ -0,0 +1,55 @@
+package tests
+
+import (
+	"github.com/Fantom-foundation/go-opera/ethapi"
+	"github.com/Fantom-foundation/go-opera/tests/contracts/transientstorage"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/holiman/uint256"
+	"testing"
+)
+
+func TestGetAccount(t *testing.T) {
+	net, err := StartIntegrationTestNet(t.TempDir())
+	if err != nil {
+		t.Fatalf("Failed to start the fake network: %v", err)
+	}
+	defer net.Stop()
+
+	// Deploy the transient storage contract
+	_, deployReceipt, err := DeployContract(net, transientstorage.DeployTransientstorage)
+	if err != nil {
+		t.Fatalf("failed to deploy contract; %v", err)
+	}
+
+	addr := deployReceipt.ContractAddress
+
+	c, err := net.GetClient()
+	if err != nil {
+		t.Fatalf("failed to get client; %v", err)
+	}
+
+	rpcClient := c.Client()
+
+	var res ethapi.GetAccountResult
+	err = rpcClient.Call(&res, "eth_getAccount", addr, rpc.LatestBlockNumber)
+	if err != nil {
+		t.Fatalf("failed to get account; %v", err)
+	}
+
+	if res.CodeHash == (common.Hash{}) {
+		t.Error("code hash should not be empty")
+	}
+
+	if res.StorageRoot == (common.Hash{}) {
```

Comment:
It is possible to use the ethereum client and the `getProof` RPC function to query for an accounts storage root using a few lines of code. An extensive example also verifying proofs can be found [here](https://github.com/Fantom-foundation/Sonic/blob/1933d89b40f511df7a8481d19b8ccedd5f19e383/tests/block_header_test.go#L702-L765). A subset of those lines should be sufficient to obtain the expected storage root hash.